app/models/user.rb

```
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  devise :database_authenticatable, :registerable, :recoverable, :rememberable, :validatable
+
+  validates :name, presence: true, length: { maximum: 225 }
+
+  before_validation :set_name, on: :create
+
+  mount_uploader :avatar, AvatarUploader
+
+  private
+
+  def set_name
+    self.name = "MovieMan №#{rand(777)}" if name.blank?
+  end
```

throw an error instead of setting default name